pkg/controller/worker/machines.go

```diff
@@ -188,6 +193,8 @@ func (w *workerDelegate) generateMachineConfig(ctx context.Context) error {
 				"cloudConfig": string(pool.UserData),
 				"kubeconfig":  string(kubeconfig),
 			},
+			"dnsPolicy": workerConfig.DNSPolicy,
+			"dnsConfig": workerConfig.DNSConfig,
```

Comment:
I don't see these values used anywhere in the machineclass Helm charts? I don't think adding them here would have any effect without it.

Reply:
you're right, good catch.